data/daftar-isi.js

```diff
@@ -8,7 +8,7 @@ export default [
         api: '/api/pancasila',
       },
       {
-        text: 'UUD 1945',
+        text: 'Pembukaan UUD 1945',
         href: '/uud1945/',
```

Comment:
Do we also need to change the slug?

Reply:
aaa right! wait a minute, I'll fix it too